Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoHardCodedPathsAnalyzerTest.cs

```diff
@@ -75,7 +75,25 @@ public void Test()
 
 		[TestMethod]
 		[TestCategory(TestDefinitions.UnitTests)]
-		public async Task CatchesHardCodedPaths2Async()
+		public async Task CatchesHardCodedPathsAsUnc()
```

Comment:
Did you intend to remove the "Async" suffix?

Reply:
Yes, in test methods this postfix doesn't add anything in my opinion

Reply:
And this particular example, I added UNC as postfix, to indicate the kind of path that is being checked.